tests/GraphQLServer.integration.test.ts

```diff
@@ -373,14 +376,44 @@ test('Should get data response if query with unknown field is executed ' +
         debug: true,
         removeValidationRecommendations: true,
         defaultValidationRules: [],
-        customValidationRules: [] 
+        customValidationRules: []
     })
     const response = await fetchResponse(`{"query":"${usersQueryWithUnknownField}"}`)
     const responseObject = await response.json()
     expect(responseObject.data.users).toStrictEqual([userOne, userTwo])
     customGraphQLServer.setOptions(INITIAL_GRAPHQL_SERVER_OPTIONS)
 })
 
+test('Should reassign AggregateError to original errors field' +
+    ' when reassignAggregateError is enabled', async() => {
```

Comment:
We should also have a test case for deactivated `reassignAggregateError`

Reply:
Added test for disabled reassignAggregateError and extracted enabled test to own class to avoid flaky test setup.